srw_handler.py

```diff
@@ -2,9 +2,11 @@
 import uti_plot_com as srw_io
 
 
-def read_srw_file(filename):
+def read_srw_file(filename, ndim=2):
```

Comment:
@danielballan, we need to dynamically pass the number of dimensions to the handler defined below (`SRWFileHandler`), but the problem is that it's registered right after the db is defined in re_config.py, so I'm not sure what's the best way to make this choice dynamic. Any ideas?

Reply:
Can you get away with registering it later?

Reply:
I think we can. Should we define 2 separate handlers?

Reply:
Seems right. You could use a factory function, but two separate classes are probably easier to document/understand quickly.

Reply:
OK, thanks @danielballan. I'll review it later.